setup.py

```diff
@@ -36,9 +36,9 @@
         'Sphinx>=1.6.3,<5.0.0',
         'sphinx-rtd-theme>=0.5.0,<0.6',
     ],
-    'dev': [
-        'compile-scss~=1.0.1',
-    ],
+    'dev': [ ],
+    #     'compile-scss~=1.0.1', NOTE: This package does not exist anymore, need to find a replacement!
```

Comment:
Should we create a story for it to not forget it?

Reply:
done: https://github.com/SwissDataScienceCenter/renku-sphinx-theme/issues/10